src/main/java/org/apache/commons/io/input/AutoCloseInputStream.java

```diff
@@ -76,7 +76,7 @@ protected void afterRead(final int n) throws IOException {
      */
     @Override
     public void close() throws IOException {
-        in.close();
+        super.close();
```

Comment:
Right, had a look at the source code, and I think:

- before: `in.close()` (removed above) could throw any `IOException` directly
- after: `super.close()` will call `ProxyInputStream#close()` which will then call `TaggedInputStream#handleIOException` (instead of `ProxyInputStream#handleIOException`)

So all good here :+1: